AnnoyancesFilter/sections/popups.txt

```diff
@@ -124,6 +124,7 @@ pcland.hu##.HMi-popup-overlay
 dictionary.com##.marketing-message
 thesaurus.com##div[class*="-RightMarketingSlot"]
 gastronom.ru##.js-bother
+news.noteru.com##.popup
```

Comment:
It is push notification, it's better to block script which is created it.
And also block gogool.ru, push management service

Reply:
yep. Add to `/firebase-messaging.js` rule